modules/planning/common/trajectory_stitcher.cc

```diff
@@ -57,9 +57,10 @@ std::vector<TrajectoryPoint>
 TrajectoryStitcher::ComputeReinitStitchingTrajectory(
     const double planning_cycle_time, const VehicleState& vehicle_state) {
   TrajectoryPoint reinit_point;
-  constexpr double kEpsilon = 0.2;
```

Comment:
@kevin-y-wang Please add a TODO here, if localization publish corrected IMU acceleration data in both modes, we should decrease this to a smaller value.

Reply:
Fixed. Thanks